bokeh/command/bootstrap.py

```diff
@@ -110,9 +112,17 @@ def main(argv):
 
     args = parser.parse_args(argv[1:])
     try:
-        args.invoke(args)
+        ret = args.invoke(args)
     except Exception as e:
-        die("ERROR: " + str(e))
+        if settings.dev:
+            raise
+        else:
+            die("ERROR: " + str(e))
+
+    if ret is False:
+        sys.exit(1)
+    elif ret is not True and isinstance(ret, int) and ret != 0:
+        sys.exit(ret)
```

Comment:
Let's just make it conventional that all subcommand `invoke` functions return integer status so this awkwardness can be avoided.

Reply:
A bit of the complexity comes from this non-sense:
```
In [1]: isinstance(True, int)                                                                                                                
Out[1]: True
```
Even if we don't proceed with this, we still need differentiate `None` from integers for backwards compatibility. I also didn't want to modify every `invoke()` implementation.